bug-bounty-hof/client.yml

```diff
@@ -1,4 +1,123 @@
 names:
+- name: David Klein
+  date: 2022-09-21
+- name: Meet Narkhede
+  date: 2022-09-20
+- name: Mario E. Mejia
+  date: 2022-09-20
+  twitter: "@Moblig_"
+- name: Anonymous
+  date: 2022-09-12
+  twitter: "@r3dpars3c"
+- name: Codermak
+  date: 2022-08-30
+  twitter: "@arshadkazmi42"
+- name: Muhammad R. Maulana
+  date: 2022-08-22
+  twitter: "@agamimaulana"
+- name: musicDespiteEverything
+  date: 2022-08-22
+- name: Irwan
+  date: 2022-08-01
+- name: Brandon Roldan
+  date: 2022-07-11
+  twitter: "@tomorrowisnew"
+  url: https://twitter.com/@tomorrowisnew
```

Comment:
We've got a mixture of entries with only a `twitter:` line, and some like this one that also have a `url:` line that also points to the person's twitter handle. Should they all have urls, or none of them? Or is there actually an important difference in how the HoF gets formatted?

Reply:
There is no difference between a user with a twitter entry and a user with a twitter entry and a url pointing to their twitter.  As for why that happens, I think it's an artifact from how we re-fill data for someone we've seen already vs handling someone we haven't seen.  Because there's no difference I never bothered trying to resolve it.